app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RestaurantsController < ActionController::Base

```

Comment:
🔨 @joni88  indentation is off, each level of indentation should only be 2 spaces